css/styles.css

```diff
@@ -533,6 +547,10 @@ hr {
 }
 
 @media only screen and (min-width: 768px) {
```

Comment:
- [ ] In the about me section, please add a distance below the button to be similar to the Figma template 👇🏽 
- [ ] In the skills section notice that the text is left aligned, you could use `align-items: flex-start;` in the class .col-center. Kindly be careful that the mobile version still working as expected. 
- [ ] Please 🙏🏽 align the title in the contact form to look like the Figma template, you could use `align-items: flex-start;` in the class .container-3.
![image](https://user-images.githubusercontent.com/58642949/188250934-e12137d9-d3a0-45e1-9b3f-0253789778af.png)

![image](https://user-images.githubusercontent.com/58642949/188251137-6e0474f9-05cd-4d3d-bb96-4c38271cb416.png)

- Your design
![image](https://user-images.githubusercontent.com/58642949/188250946-bc70ad37-cf85-4762-9961-0f86abd97e90.png)
![image](https://user-images.githubusercontent.com/58642949/188251160-4ab3353d-3a1b-4e90-8fe6-f4dc7bc28752.png)